docs/js/custom.js

```
@@ -0,0 +1,140 @@
+        const codeElem = module.querySelector("h2 code");
+        if (codeElem) {
+            codeElem.style.color = "black";
+            codeElem.style.border = "none";
+        }
+    });
+
+    // Append "class" Before Class Names
+    document.querySelectorAll(".doc.doc-object.doc-class").forEach(classElem => {
+        const codeElem = classElem.querySelector("h3 code");
+        if (codeElem) {
+            codeElem.innerHTML = "<span class='class'>class </span>" + codeElem.innerHTML;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)